pkg/cmd/cli/install/install.go

```diff
@@ -389,6 +407,25 @@ func (o *Options) Run(c *cobra.Command, f client.Factory) error {
 	if err != nil {
 		return err
 	}
+
+	// Get Kubernetes client for priority class validation
+	kubeClient, err := f.KubeClient()
+	if err != nil {
+		return err
+	}
+
+	// Validate priority classes if specified
+	logger := logrus.New()
+	logger.SetOutput(os.Stdout)
+	if o.ServerPriorityClassName != "" {
+		_ = kubeutil.ValidatePriorityClass(context.Background(), kubeClient, o.ServerPriorityClassName, logger.WithField("component", "server"))
```

Comment:
Not sure if we need the validation in this way. This will print logs on the client side, users cannot see anything in the console.
Moreover, it would not an invalid case that the priorityClass CR doesn't exist during install, but added later.